planning/development-strategy.md

```diff
@@ -29,34 +29,41 @@ A simple little website about trees.
 
 <!-- user story -->
 
```

Comment:
"as a user I can read the title of the page"
  -[ ] there is a prominent title at the top of the web page